cli/config/config.go

```diff
@@ -17,12 +16,18 @@ const K3sManifestPath = "/var/lib/rancher/k3s/server/manifests"
 const K3sImagePath = "/var/lib/rancher/k3s/agent/images"
 const PackageInitName = "zarf-init.tar.zst"
 const PackagePrefix = "zarf-package-"
-const ZarfLocalIP = "127.0.0.1"
 const ZarfGitUser = "zarf-git-user"
+const ZarfStatePath = ".zarf-state.yaml"
 
 var CLIVersion = "unset"
+var config ZarfPackage
+var state ZarfState
 
-var config ZarfConfig
+func init() {
```

Comment:
Should this be an exported function? It's not being used in this package and I think it would be useful the in cmd package for initializing?

Reply:
init()?

https://stackoverflow.com/questions/24790175/when-is-the-init-function-run

Reply:
TIL... Cool!

Reply:
![](https://media.giphy.com/media/Ri7d8I18cto2jufOKc/giphy.gif)